libmamba/ext/solv-cpp/src/solvable.cpp

```diff
@@ -206,6 +206,22 @@ namespace solv
         return set_license(str.c_str());
     }
 
+    auto ObjSolvableViewConst::python_site_packages_path() const -> std::string_view
+    {
+        return ptr_to_strview(::solvable_lookup_str(const_cast<::Solvable*>(raw()), SOLVABLE_MEDIABASE)
```

Comment:
Thank you for working on this!

One concern (raised in this [issue](https://github.com/openSUSE/libsolv/issues/579)) would be that `SOLVABLE_MEDIABASE` seems to be used for the channel [here](https://github.com/mamba-org/mamba/blob/main/libmamba/ext/solv-cpp/src/repo.cpp#L309-L317).
`channel` and `set_channel` don't seem to be used though other than in the tests, but still not sure how using that placeholder could interfere with what `libsolv` could be doing...
Maybe @AntoinePrv has more insight?

Reply:
Good catch! Basically for most attribute we use (other name name, version...), this is only a key:value store and has little importance which key we use (the point of these methods is to hide them as implementation details).

I think since for the channel it is used with another struct, it does not matter. Alternatively one could use another obscure key with proper comment.